hca_ingest/api/ingestapi.py

```diff
@@ -132,12 +150,12 @@ def get_resource_repository_url(self, repository_name: str):
             return self._ingest_links[repository_name]["href"].rsplit("{")[0]
         return None
 
-    def getSubmissions(self):
+    def get_submissions(self):
```

Comment:
Who is using this function?
See [`getSubmissions` text instances in our repo](https://github.com/search?q=org%3Aebi-ait+getSubmissions&type=code)

Reply:
Nothing is using this, which is why I changed it to match the python naming conventions of the rest of our methods